integrationtests/security-it/src/test/java/org/infinispan/test/integration/security/embedded/NodeAuthPropertiesHandler.java

```diff
@@ -0,0 +1,116 @@
+
+ package org.infinispan.test.integration.security.embedded;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.infinispan.Cache;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.test.integration.security.utils.Deployments;
+import org.infinispan.util.logging.Log;
+import org.infinispan.util.logging.LogFactory;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.OperateOnDeployment;
+import org.jboss.arquillian.container.test.api.TargetsContainer;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.junit.InSequence;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * 
+ * @author vjuranek
+ * @since 8.0
+ */
+@RunWith(Arquillian.class)
+public class NodeAuthPropertiesHandler extends AbstractNodeAuthentication {
```

Comment:
Could you rename the test to have an IT at the end? Please also add a blurb in the javadoc.